src/signalingclient/DefaultSignalingClient.ts

```diff
@@ -437,16 +437,16 @@ export default class DefaultSignalingClient implements SignalingClient {
     this.webSocket.addEventListener('close', this.closeEventHandler);
     this.webSocket.addEventListener('error', () => {
       if (this.isClosing && !this.wasOpened) {
-        this.logger.info('ignoring error closing signaling while connecting');
+        this.logger.info('Signaling WebSocket ignoring error closing while connecting');
         return;
       }
       if (this.wasOpened) {
-        this.logger.error('received error while connected');
+        this.logger.error('Signaling WebSocket received error while connected');
```

Comment:
Should there be a : `Signaling WebSocket: Received error while connected`?

Reply:
Either works, but my personal preference is for log lines that can be read as close to a 'plain english' sentence. I wasn't trying to create a searchable log prefix here (on that note...I tend to not like manually added log prefixes as there is no enforcement mechanism and they are often added inconsistently; we should eventually refactor our logger to support additional attributes)